lib/tokens.js

```diff
@@ -442,6 +452,11 @@ export async function resolve({requester, token, levelOfAssurance}) {
         });
     }
 
+    if(tokenRecord) {
```

Comment:
```suggestion
    if(tokenRecord) {
      // token was previously resolved and we now know it hasn't been invalidated,
      // so now it can be returned
```